lib/active_model/serializable_resource.rb

```diff
@@ -74,6 +75,14 @@ def serializer?
       use_adapter? && !!(serializer)
     end
 
+    def instrument
```

Comment:
Hi @bf4 thanks for te review.
Could you please provide more info about it? What kind of integration you are thinking? Maybe you can update the #1089 with more info.